dotnet/src/Extensions/PromptTemplates.Handlebars/Helpers/KernelHelpers/KernelFunctionHelpers.cs

```diff
@@ -211,7 +211,7 @@ private static void ProcessPositionalArguments(KernelFunctionMetadata functionMe
         if (resultAsObject is RestApiOperationResponse restApiOperationResponse)
         {
             // Deserialize any JSON content or return the content as a string
-            if (string.Equals(restApiOperationResponse.ContentType, "application/json", StringComparison.OrdinalIgnoreCase))
+            if (restApiOperationResponse.ContentType.IndexOf("application/json", StringComparison.OrdinalIgnoreCase) > 0)
```

Comment:
Why not just use 
if (restApiOperationResponse.ContentType.Contains("application/json"))
???

Reply:
Something about this project doesn't recognize `public bool Contains(String value, StringComparison comparisonType);` and using Contains without the StringComparison parameter will cause a linter error